src/main/java/com/google/sps/data/HTTPStatusCodeException.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import java.io.BufferedReader;
```

Comment:
Are all these imports needed here?
Only import what the code needs.

Reply:
ah, ok!